sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -405,13 +405,16 @@ def _add_argparse_args(cls, parser):
     parser.add_argument('--template_location',
                         default=None,
                         help='Save job to specified local or GCS location.')
-    parser.add_argument(
-        '--label', '--labels',
-        dest='labels',
-        action='append',
-        default=None,
-        help='Labels that will be applied to this Dataflow job. Labels are key '
-        'value pairs separated by = (e.g. --label key=value).')
+    parser.add_argument('--label', '--labels',
+                        dest='labels',
+                        action='append',
+                        default=None,
+                        help='Labels that will be applied to this Dataflow job.'
+                        'Labels are key value pairs separated by = '
+                        '(e.g. --label key=value).')
+    parser.add_argument('--update',
+                        default=False,
+                        action='store_true')
```

Comment:
Perhaps a help string to clarify that this experimental would help until we have an integration test.

Reply:
Done.